packages/fxa-auth-server/test/local/routes/subscriptions/stripe.js

```diff
@@ -617,9 +618,36 @@ describe('DirectStripeRoutes', () => {
       const actual = await directStripeRoutesInstance.createCustomer(
         VALID_REQUEST
       );
+      const callArgs =
+        directStripeRoutesInstance.stripeHelper.createPlainCustomer.getCall(
+          0
+        ).args;
+      assert.equal(callArgs[4], undefined);
 
       assert.deepEqual(filterCustomer(expected), actual);
     });
+
+    it('creates a stripe customer with the ip address on automatic tax', async () => {
+      const expected = deepCopy(emptyCustomer);
+      directStripeRoutesInstance.stripeHelper.createPlainCustomer.resolves(
```

Comment:
I don't see an updated test for `createPlainCustomer` to assert it calls `stripeHelper.stripe.customers.create` with the expected args (now including the `tax` field with the `ipAddress`)?

Reply:
Nice catch! Updated with a test.